packages/graph/hash_graph/lib/graph/src/knowledge/link/mod.rs

```diff
@@ -55,34 +55,6 @@ impl fmt::Display for Link {
     }
 }
 
-// TODO: Add PersistedLink to expose metadata about link instances
-
-/// Outgoing link targets of a source entity sharing the same link type. May be one or many
-/// [`EntityId`]s
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(untagged)]
-pub enum OutgoingLinkTarget {
-    Single(EntityId),
-    Multiple(Vec<EntityId>),
-}
-
-/// A collection of links that originate from the same source entity.
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
-#[serde(rename_all = "camelCase")]
-pub struct OutgoingLinks(HashMap<VersionedUri, OutgoingLinkTarget>);
-
-impl OutgoingLinks {
-    #[must_use]
-    pub const fn new(links: HashMap<VersionedUri, OutgoingLinkTarget>) -> Self {
-        Self(links)
-    }
-
-    #[must_use]
-    pub const fn outgoing(&self) -> &HashMap<VersionedUri, OutgoingLinkTarget> {
-        &self.0
-    }
-}
-
 /// Specifies whether or not a link is active.
 #[derive(Debug, Clone, Copy)]
 #[non_exhaustive]
```

Comment:
I'll get rid of this struct soon as well. Will make this module very short :tada: